net-p2p/bitcoin-qt/bitcoin-qt-0.13.1.ebuild

```diff
@@ -0,0 +1,130 @@
+# Copyright 2010-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=5
+
+BITCOINCORE_COMMITHASH="03422e564b552c1d3c16ae854f8471f7cb39e25d"
+BITCOINCORE_LJR_DATE="20161027"
+BITCOINCORE_IUSE="dbus kde +libevent ljr +qrcode qt4 qt5 +http test +tor upnp +wallet zeromq"
+BITCOINCORE_POLICY_PATCHES="+rbf spamfilter"
+LANGS="af af_ZA ar be_BY bg bg_BG ca ca@valencia ca_ES cs cs_CZ cy da de el el_GR en en_GB eo es es_AR es_CL es_CO es_DO es_ES es_MX es_UY es_VE et eu_ES fa fa_IR fi fr fr_CA fr_FR gl he hi_IN hr hu id_ID it it_IT ja ka kk_KZ ko_KR ku_IQ ky la lt lv_LV mk_MK mn ms_MY nb ne nl nl_NL pam pl pt_BR pt_PT ro ro_RO ru ru_RU sk sl_SI sq sr sr@latin sv ta th_TH tr tr_TR uk ur_PK uz@Cyrl vi vi_VN zh zh_CN zh_HK zh_TW"
+KNOTS_LANGS="nl_NL"
+BITCOINCORE_NEED_LEVELDB=1
+BITCOINCORE_NEED_LIBSECP256K1=1
+inherit bitcoincore eutils fdo-mime gnome2-utils kde4-functions qt4-r2
+
+DESCRIPTION="An end-user Qt GUI for the Bitcoin crypto-currency"
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~arm64 ~mips ~ppc ~x86 ~amd64-linux ~x86-linux"
+
+RDEPEND="
+	dev-libs/protobuf
+	qrcode? (
+		media-gfx/qrencode
+	)
+	qt4? ( dev-qt/qtcore:4[ssl] dev-qt/qtgui:4 )
+	qt5? ( dev-qt/qtgui:5 dev-qt/qtnetwork:5 dev-qt/qtwidgets:5 )
+	dbus? (
+		qt4? ( dev-qt/qtdbus:4 )
+		qt5? ( dev-qt/qtdbus:5 )
+	)
+"
+DEPEND="${RDEPEND}
+	qt5? ( dev-qt/linguist-tools:5 )
+	ljr? (
+		gnome-base/librsvg
+		media-gfx/imagemagick[png]
+	)
+"
+REQUIRED_USE="^^ ( qt4 qt5 )
```

Comment:
For reference: https://wiki.gentoo.org/wiki/Project:Qt/Policies#Handling_different_versions_of_Qt